blogs/2016/10/31/js_roundup_2.md

```diff
@@ -0,0 +1,133 @@
+
+Joel Day has built an exceptionally useful extension for writing JavaScript and TypeScript docs. This extension will automatically generates detailed JSDoc comments for your code. 
+
+![document this](2016_10_31_document_this.gif)
+
+## Beautify 
+
+Marketplace - [Beautify](https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify)
+
+Publisher - [HookyQR](https://marketplace.visualstudio.com/search?term=publisher%3A%22HookyQR%22&target=VSCode)
+
+Internally, VS Code uses [js-beautify](https://www.npmjs.com/package/js-beautify). This extension allows you to specify a `.jsbeautifyrc` file to indicate the formatting style for your JavaScript, CSS, Sass, and HTML code. 
+
+You can search the Marketplace for more formatters using the new [Formatters category](https://marketplace.visualstudio.com/search?target=VSCode&category=Formatters&sortBy=Downloads). 
+
+>**Note:** For extension authors, we are working on a blog post for Formatting best practices. Stay tuned as it will be coming soon. 
+
+# Keymaps for Sublime Text and Atom
+
+There are two extensions in this section, one for [Atom](https://marketplace.visualstudio.com/items?itemName=ms-vscode.atom-keybindings) and another for [Sublime Text](https://marketplace.visualstudio.com/items?itemName=ms-vscode.sublime-keybindings). If you have used these editors you have likely developed habits with your keyboards shortcuts. These extensions bring the keyboard shortcuts from Atom and Sublime Text to VS Code. 
+
+These extensions are in preview because we want your feedback. There are still many keyboard shortcuts to add. Adding keyboard shortcuts we missed is easy. 
+
+1. Go to the extension's GitHub repository ([Atom](https://github.com/waderyan/vscode-atom-keybindings) and [Sublime Text](https://github.com/Microsoft/vscode-sublime-keybindings))
+2. Open package.json ([Atom](https://github.com/waderyan/vscode-atom-keybindings/blob/master/package.json) and [Sublime Text](https://github.com/Microsoft/vscode-sublime-keybindings/blob/master/package.json)).
+3. Add a JSON object to `contributions.keybindings` section of `package.json` as seen below ([Atom](https://github.com/waderyan/vscode-atom-keybindings/blob/master/package.json#L25) and [Sublime Text](https://github.com/Microsoft/vscode-sublime-keybindings/blob/master/package.json#L25)).
+4. Open a pull request.
+
+```json
+{
+    "mac": "<keyboard shortcut for mac>",
+    "linux": "<keyboard shortcut for linux",
+    "win": "<keyboard shortcut for windows",
+    "key": "<default keyboard shortcut>",
+    "command": "<name of the command in VS Code"
+}
+```
+
+If you find that VS Code does not have a feature from Atom or Sublime Text, please comment in these Github issues ([Atom](https://github.com/waderyan/vscode/issues/14316) and [Sublime Text](https://github.com/Microsoft/vscode/issues/3776)). Additionally, there are many extensions in the Marketplace to add the missing features. 
+
```

Comment:
Give this another H2 header. Something like "Extension grab bag" or "It's raining extensions".